lib/computer.rb

```diff
@@ -2,7 +2,7 @@ class Computer
   attr_reader :board
 
   def initialize
-    @board = (Board.new)
+    @board = Board.new
     @taken_coordinates = []
```

Comment:
I like it. do you think we'll use this instance variable to help assess if the player or computer has lost? for instance, when the ship is sunk, we could take the coordinate out of that array. I feel that might not work, but its a thought